### PR TITLE
Delimiter has been changed from . to $

### PR DESCRIPTION
**Pull request by Yumfriez**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -259,11 +259,11 @@ private static void copyFromUrl(GString localScriptPath, String scriptUrl) {
 
 task copyTestDatabaseScripts {
     def initScriptPath = "${projectDir}/src/test/resources/db/migration/V001__initialize_schema.sql"
-    def initScriptUrl = "https://raw.githubusercontent.com/reportportal/migrations/master/migrations/1_initialize_schema.up.sql"
+    def initScriptUrl = "https://raw.githubusercontent.com/reportportal/migrations/project-attributes-fix-db/migrations/1_initialize_schema.up.sql"
```
Comment (ihar-kahadouski) [inline build.gradle]: It should not be in ps-migrations branch

File: build.gradle [hunk 2]
```diff
@@ -259,11 +259,11 @@ private static void copyFromUrl(GString localScriptPath, String scriptUrl) {
...
     copyFromUrl(initScriptPath, initScriptUrl)
 
     def dataScriptPath = "${projectDir}/src/test/resources/db/migration/V002__default_data.sql"
-    def dataScriptUrl = "https://raw.githubusercontent.com/reportportal/migrations/master/migrations/3_default_data.up.sql"
+    def dataScriptUrl = "https://raw.githubusercontent.com/reportportal/migrations/project-attributes-fix-db/migrations/3_default_data.up.sql"
```
Comment (ihar-kahadouski) [inline build.gradle]: same

File: src/main/resources/demo/attachments/har.har [hunk 3]
```diff
@@ -5084,7 +5084,7 @@
...
...\"}],\"PRODUCT_BUG\":[{\"id\":3,\"locator\":\"pb001\",\"typeRef\":\"PRODUCT_BUG\",\"longName\":\"Product Bug\",\"shortName\":\"PB\",\"color\":\"#ec3900\"}],\"SYSTEM_ISSUE\":[{\"id\":5,\"locator\":\"si001\",\"typeRef\":\"SYSTEM_ISSUE\",\"longName\":\"System Issue\",\"shortName\":\"SI\",\"color\":\"#0274d1\"}]},\"notificationsConfiguration\":{\"enabled\":false,\"cases\":[{\"recipients\":[\"OWNER\"],\"sendCase\":\"always\",\"launchNames\":[],\"attributes\":[]}]}},\"users\":[{\"login\":\"default\",\"projectRole\":\"PROJECT_MANAGER\"},{\"login\":\"kolya\",\"projectRole\":\"OPERATOR\"},{\"login\":\"petya\",\"projectRole\":\"CUSTOMER\"},{\"login\":\"vasya\",\"projectRole\":\"MEMBER\"},{\"login\":\"roma\",\"projectRole\":\"PROJECT_MANAGER\"},{\"login\":\"superadmin\",\"projectRole\":\"MEMBER\"}],\"integrations\":[{\"id\":5,\"projectId\":2,\"integrationType\":{\"enabled\":true,\"type\":4,\"name\":\"email\",\"isEnabled\":true,\"creationDate\":1553072627019,\"groupType\":\"NOTIFICATION\"},\"integrationParameters\":{\"rules\":[{\"recipients\":[\"owner\"]},{\"launchStatsRule\":\"always\"}]},\"enabled\":false,\"creationDate\":1553072627019}],\"organization\":null,\"creationDate\":1553072627019}"
```
Comment (ihar-kahadouski) [inline src/main/resources/demo/attachments/har.har]: it is not necessary change it here

File: src/main/java/com/epam/ta/reportportal/ws/converter/converters/ProjectConverter.java [hunk 4]
```diff
@@ -43,7 +43,7 @@
 @Service
 public final class ProjectConverter {
 
-	private final static String INDEXING_RUN = "analyzer.indexingRunning";
+	private final static String INDEXING_RUN = "analyzer$indexingRunning";
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/ws/converter/converters/ProjectConverter.java]: maybe take it from the project attribute enum?

**Pull request closed without merging**